packages/react-components/src/components/UnsupportedOperatingSystem.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(unsupported-browser) */
+import { Icon, Link, Stack, Text } from '@fluentui/react';
+/* @conditional-compile-remove(unsupported-browser) */
+import { _pxToRem } from '@internal/acs-ui-common';
+import React from 'react';
+/* @conditional-compile-remove(unsupported-browser) */
+import { useLocale } from '../localization';
+/* @conditional-compile-remove(unsupported-browser) */
+import {
+  containerStyles,
+  iconStyles,
+  linkTextStyles,
+  mainTextStyles,
+  secondaryTextStyles
+} from './styles/UnsupportedEnvironment.styles';
+
+/**
+ * Strings for UnsupportedBrowser component
+ *
+ * @beta
+ */
+export interface UnsupportedOperatingSystemStrings {
+  /** String for the primary text */
+  primaryText: string;
+  /** String for the secondary text */
+  secondaryText: string;
+  /** String for the help link */
+  moreHelpLink: string;
+}
+
+/**
+ * props for UnsupportedOperatingSystem UI
+ *
+ * @beta
+ */
+export interface UnsupportedOperatingSystemProps {
+  /** Handler to perform a action when the help link is actioned */
+  onTroubleshootingClick?: () => void;
+  /** String overrides for the component */
+  strings: UnsupportedOperatingSystemStrings;
+}
+
+/* @conditional-compile-remove(unsupported-browser) */
+const UnsupportedOperatingSystemContainer = (props: UnsupportedOperatingSystemProps): JSX.Element => {
+  const { onTroubleshootingClick, strings } = props;
+  return (
+    <Stack styles={containerStyles}>
+      <Icon styles={iconStyles} iconName="UnsupportedBrowserWarning" data-ui-id="unsupportedBrowserIcon"></Icon>
```

Comment:
Should these also be update
```suggestion
      <Icon styles={iconStyles} iconName="UnsupportedOperatingSystemWarning" data-ui-id="unsupportedOSIcon"></Icon>
```